.travis.yml

```diff
@@ -14,15 +14,20 @@ release_job: &release_job
 jobs:
   include:
     - stage: test
+      env:
+        # Path to 'hadolint' binary
+        HADOLINT: "${HOME}/hadolint"
```

Comment:
I don't think so, does `~/hadolint` have a benefit over this?

Reply:
`env` adds entries to the build matrix so it will show up in the UI as a label for the job, which is a bit confusing because it is not really a matrix entry but just a convenience variable